### ISSUE TITLE
[BUG] .NET runtime error and an exception was thrown on the constructor call, resulting in a faulty KERNELBASE.dll that will not start.

### ISSUE THREAD

**Issue by ONMYBEAT** [Alkl58/NotEnoughAV1Encodes#154]
**Describe the bug**
Batch processing was stopped in the middle of a process, and after restarting the program, it no longer started.

**To Reproduce**
I can't activate it even if I wanted to reproduce it.

**Expected behavior**
I need a fix or workaround.

**Screenshots**
It will not start and I cannot take a screenshot.

**Log File**
Submit two error logs in the Event Viewer.
One is a .net runtime error, the other is an Application error.

**Desktop (please complete the following information):**
 - OS: Windows10 Pro 22H2
 - NEAV1E Version: 2.1.5
 - 
[eventlog.zip](https://github.com/Alkl58/NotEnoughAV1Encodes/files/13953549/eventlog.zip)

**Comment by ONMYBEAT** [comment]
PS
NEAV1E version 2 will happen in all cases. However, with version 1.9, it is possible to start the program without any problem.

**Comment by ONMYBEAT** [comment]
I didn't get a reply, but the accident was resolved.
I deleted the data in %%AppData%\Roaming\NEAV1E and it started. I would like to have this data deleted when uninstalling.
I didn't notice it at all.

**Comment by Alkl58** [comment]
I was able to reproduce a similar crash by manually corrupting a queue item (json file).

The fix I pushed is only there to catch the crash and not the reason why the queue item corrupted in the first place.

Thanks to your event logs, the issue was clearly:
```
Newtonsoft.Json.JsonReaderException: Unterminated string. Expected delimiter: ". Path 'ChunkProgress[196]', line 1008, position 7.
```

I can only guess that maybe restarting the program while it was writing to the file may have corrupted it. Not sure.

If anyone encounters this issue, I would greatly appreciate getting my hands on the corrupted json file located under `%appdata%\NEAV1E\Queue`.

I will also look into how I can make the uninstaller clean up the leftovers.
